bin/daemon.py

```diff
@@ -727,9 +728,9 @@ def is_socket(fd):
         """
     result = False
 
-    file_socket = socket.fromfd(fd, socket.AF_INET, socket.SOCK_RAW)
 
```

Comment:
Nit: if you could delete one of the 2 empty lines it would be a bit nicer

Reply:
Thanks, fixed.
Also reworeded the comment at the top.